package.json

```diff
@@ -53,34 +71,53 @@
     "karma-tap-pretty-reporter": "4.2.0",
     "karma-webpack": "git://github.com/ryanclark/karma-webpack#eec6dfc",
     "kw-doc": "2.0.0",
-    "kw-web-suite": "11.0.0",
+    "moment": "2.29.1",
```

Comment:
Do we use moment?

Reply:
We use it in [`webpack.prod.js`](https://github.com/Kitware/vtk-js/blob/master/webpack.prod.js#L3).